src/App.jsx

```diff
@@ -0,0 +1,49 @@
+import "./App.css";
+import { useState } from "react";
+import { useEffect } from "react";
+import divider from "./assets/divider.svg";
+import dice from "./assets/dice.svg";
+
+function App() {
+  const [adviceNumber, setAdviceNumber] = useState(null);
+  const [adviceText, setAdviceText] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const getAdvice = async () => {
+    try {
+      setIsLoading(true);
+
+      const response = await fetch("https://api.adviceslip.com/advice");
```

Comment:
In this case API URL is public, but if it was private, we'd have to hide it, 

Take these steps to practice working with environment variables 👇

1. Create a `.env` file in the top level of the project, following the template from `.env.example` if there is one.
2. Put the variable in `.env`, use names starting with `VITE_`, e.g. `VITE_API_URL=https://api.adviceslip.com/advice`
3. Use your variables inside your code like below 

```suggestion
      const response = await fetch(import.meta.env.VITE_API_URL);
```


Reference from Vite: [Env Variables and Modes](https://vitejs.dev/guide/env-and-mode.html)